### ISSUE TITLE
Installer does not detect systemd on Ubuntu 14.04

### ISSUE THREAD

**Issue by Nani-o** [netdata/netdata#4421]
Hi,

There is what seems to be a bug in the installer, the issytemd function has been updated the 11th September of 2018 with this commit : [baac9dc53bc204d4506d7b36b190059d4685bd57](
https://github.com/netdata/netdata/commit/baac9dc53bc204d4506d7b36b190059d4685bd57#diff-73dacc6abc355a242efa2c913233f934)

As you can see, a check has been removed, but this check was also setting up a variable that is used later to loop on : 

```SHELL
    # -- ubuntu runs systemd as init ---
    ## if systemd is not running, it is not systemd
    #pids=$(pidof systemd 2>/dev/null)
    #[ -z "${pids}" ] && return 1

    # check if the running systemd processes are not in our namespace
    myns="$(readlink /proc/self/ns/pid 2>/dev/null)"
    for p in ${pids}
    do
        ns="$(readlink /proc/${p}/ns/pid 2>/dev/null)"

        # if pid of systemd is in our namespace, it is systemd
        [ ! -z "${myns}" ] && [ "${myns}" = "${ns}" ] && return 0
    done

    # else, it is not systemd
    return 1
}

**Comment by ktsaou** [comment]
hm... this seems a bug.
It was introduced with this: https://github.com/netdata/netdata/pull/4151

Let me revert it.

**Comment by ktsaou** [comment]
fixed in #4423

**Comment by ktsaou** [comment]
merged it.
Thank you for reporting it.
